src/main/java/com/amazonaws/services/neptune/cluster/AddCloneTask.java

```diff
@@ -397,9 +397,11 @@ private void createInstance(String name,
             } catch (NeptuneException e) {
                 // Check if we've exhausted our retries
                 if (attempt == maxRetries) {
-                    logger.error("Failed to create {} instance after {} attempts", name, maxRetries);
-                    throw e;
+                    logger.error("Failed to create {} instance after {} attempts, with error {}", name, maxRetries, e.getMessage());
```

Comment:
Should this be warn? I'm fine either way, since this is actually an error

Reply:
I prefer an error. I want it to really stand out in the logs as something has gone seriously wrong, although I believe it is recoverable enough that it's worth allowing the export to proceed.